### DISCUSSION TITLE
Changing repositories to bookworm

### DISCUSSION THREAD

**Discussion by peerchemist** [SpiralLinux/SpiralLinux-project#232] [Q&A]
Hello, 

I have been testing this lovely distro in a virtual machine. Specifically, I wanted to check out the workflow of snapper and migrating to debian-testing (bookworm). I've tried twice to change the repository from buster to bookworm, however, both times system update has resulted in system breakage. 

Any tips?

**Comment by methanoid** [comment]
Worked fine for me yesterday, literally just replacing every mention of bullseye with bookworm and doing a system update.

Very slow tho...

**Comment by methanoid** [comment]
@geckolinux given Bookworm release is VERY close is it possible for you to update the build scripts and maybe release "testing" ISOs of Bookworm (or the revised archives so we can run the build ourselves)? I don't feel totally confident in editing the build scripts to make my own (I'm sure I'd miss something).  

Yesterday's upgrade to Bookworm with new kernels, new KDE 5.27 (guess which version I was running!) etc looks really nice.

**Comment by Reyjfb** [comment]
Question for op: did you follow the procedure to change repos that is listed on the site? I ask because I have changed repos on 6 different machines from stable to Sid without issues
